miqi/bridge/loop.py

```
@@ -979,7 +980,8 @@ async def _decode_and_parse(att: dict) -> tuple[str, str] | None:
                             # threads.start may win the INSERT race; IntegrityError
                             # is fine — we update metadata next anyway.
                             await thread_runtime.create_thread(title="New thread", thread_id=thread_id)
-                        except Exception:
+                        except sqlite3.IntegrityError:
+                            # 并发 threads.start 已建同 id 线程——竞争正常，忽略
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)